app/hellodjango/locations/helpers/geocoding.py

```
@@ -0,0 +1,41 @@
+    return distance.m / (111_319.5 * math.cos(lat_radians))
+
+
+def geocode_with_nominatim_public(concatenated_address):
+    """Geocode a single address via Nominatim public endpoint."""
+    try:
+        geocoder = geocoders.Nominatim(user_agent=settings.NOMINATIM_USER_AGENT)
+        result = geocoder.geocode(concatenated_address, addressdetails=True)
+        address_information = result.raw
+        logger.info(f"Successfully geocoded {concatenated_address}")
+        return address_information
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)